<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>380</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>380</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>640</width>
    <height>380</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>640</width>
      <height>380</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>640</width>
      <height>380</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>640</width>
      <height>380</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background: darkblue</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="ScoreBtn">
     <property name="geometry">
      <rect>
       <x>1</x>
       <y>140</y>
       <width>641</width>
       <height>33</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe Print</family>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border: none;
color: green;
QPushButton
{
	color: white;
}</string>
     </property>
     <property name="text">
      <string>Field Size</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ConfigBtn">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>180</y>
       <width>641</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe Print</family>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border: none;
color: green;
QPushButton
{
	color: white;
}</string>
     </property>
     <property name="text">
      <string>Game Level</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ExitBtn">
     <property name="geometry">
      <rect>
       <x>-110</x>
       <y>320</y>
       <width>641</width>
       <height>33</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe Print</family>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border: none;
color: green;
QPushButton
{
	color: white;
}</string>
     </property>
     <property name="text">
      <string>Return</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>621</width>
       <height>101</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Ravie</family>
       <pointsize>26</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: green;
</string>
     </property>
     <property name="text">
      <string>Configuration</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="ConfigBtn_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>230</y>
       <width>641</width>
       <height>33</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe Print</family>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border: none;
color: green;
QPushButton
{
	color: white;
}</string>
     </property>
     <property name="text">
      <string>Extended Game</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ConfigBtn_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>280</y>
       <width>641</width>
       <height>33</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe Print</family>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border: none;
color: green;
QPushButton
{
	color: white;
}</string>
     </property>
     <property name="text">
      <string>Player</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>150</y>
       <width>70</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_2">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>190</y>
       <width>70</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_3">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>240</y>
       <width>70</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_4">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>290</y>
       <width>70</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>10</y>
       <width>12</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Ravie</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: orange;
</string>
     </property>
     <property name="text">
      <string>e</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>14</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Ravie</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: red;
</string>
     </property>
     <property name="text">
      <string>T</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>10</y>
       <width>13</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Ravie</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: Yellow;
</string>
     </property>
     <property name="text">
      <string>t</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>10</y>
       <width>9</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Ravie</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: blue;
</string>
     </property>
     <property name="text">
      <string>i</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>10</y>
       <width>12</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Ravie</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: green;
</string>
     </property>
     <property name="text">
      <string>r</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>10</y>
       <width>12</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Ravie</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: purple;
</string>
     </property>
     <property name="text">
      <string>s</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <zorder>ConfigBtn_2</zorder>
    <zorder>ScoreBtn</zorder>
    <zorder>ConfigBtn</zorder>
    <zorder>ExitBtn</zorder>
    <zorder>label_8</zorder>
    <zorder>ConfigBtn_3</zorder>
    <zorder>checkBox_2</zorder>
    <zorder>checkBox_3</zorder>
    <zorder>checkBox_4</zorder>
    <zorder>checkBox</zorder>
    <zorder>label_9</zorder>
    <zorder>label_3</zorder>
    <zorder>label_4</zorder>
    <zorder>label_7</zorder>
    <zorder>label_5</zorder>
    <zorder>label_6</zorder>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
